app/views/layouts/react.html.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
this is a temporary file, yes?

Reply:
Yes!  We were testing integration strategies for the React pages... We will clean it up before we push the final version.